blockchain/blockchain-wallet-v4-frontend
# Fix/SelectBox error label position

## Description
Set SelectBox error position by passing props 

Close #1066

## Change Type
Please enter one or more of the following: 
- Bug Fix

## Testing Steps
Go to SFOX sign up page, triggers a form validation error in "State" dropdown select. Error label should be placed on top of the input box.

## Code Checklist
